RAKE/FoxStopList.py

```diff
@@ -0,0 +1,430 @@
+ 'herself',
+ 'here',
+ 'high',
+ 'higher',
+ 'highest',
+ 'him',
+ 'himself',
+ 'his',
+ 'how',
+ 'however',
+ 'i',
+ 'if',
+ 'important',
+ 'in',
+ 'interest',
+ 'interested',
+ 'interesting',
+ 'interests',
+ 'into',
+ 'is',
+ 'it',
+ 'its',
+ 'itself',
+ 'j',
+ 'just',
+ 'k',
+ 'keep',
+ 'keeps',
+ 'kind',
+ 'knew',
+ 'know',
+ 'known',
+ 'knows',
+ 'l',
+ 'large',
+ 'largely',
+ 'last',
+ 'later',
+ 'latest',
+ 'least',
+ 'less',
+ 'let',
+ 'lets',
+ 'like',
+ 'likely',
+ 'long',
+ 'longer',
+ 'longest',
+ 'm',
+ 'made',
+ 'make',
+ 'making',
+ 'man',
+ 'many',
+ 'may',
+ 'me',
+ 'member',
+ 'members',
+ 'men',
+ 'might',
+ 'more',
+ 'most',
+ 'mostly',
+ 'mr',
+ 'mrs',
+ 'much',
+ 'must',
+ 'my',
+ 'myself',
+ 'n',
+ 'necessary',
+ 'need',
+ 'needed',
+ 'needing',
+ 'needs',
+ 'never',
+ 'new',
+ 'newer',
+ 'newest',
+ 'next',
+ 'no',
+ 'non',
+ 'not',
+ 'nobody',
+ 'noone',
+ 'nothing',
+ 'now',
+ 'nowhere',
+ 'number',
+ 'numbered',
+ 'numbering',
+ 'numbers',
+ 'o',
+ 'of',
+ 'off',
+ 'often',
+ 'old',
+ 'older',
+ 'oldest',
+ 'on',
+ 'once',
+ 'one',
+ 'only',
+ 'open',
+ 'opened',
+ 'opening',
+ 'opens',
+ 'or',
+ 'order',
+ 'ordered',
+ 'ordering',
+ 'orders',
+ 'other',
+ 'others',
+ 'our',
+ 'out',
+ 'over',
+ 'p',
+ 'part',
+ 'parted',
+ 'parting',
+ 'parts',
+ 'per',
+ 'perhaps',
+ 'place',
+ 'places',
+ 'point',
+ 'pointed',
+ 'pointing',
+ 'points',
+ 'possible',
+ 'present',
+ 'presented',
+ 'presenting',
+ 'presents',
+ 'problem',
+ 'problems',
+ 'put',
+ 'puts',
+ 'q',
+ 'quite',
+ 'r',
+ 'rather',
+ 'really',
+ 'right',
+ 'room',
+ 'rooms',
+ 's',
+ 'said',
+ 'same',
+ 'saw',
+ 'say',
+ 'says',
+ 'second',
+ 'seconds',
+ 'see',
+ 'seem',
+ 'seemed',
+ 'seeming',
+ 'seems',
+ 'sees',
+ 'several',
+ 'shall',
+ 'she',
+ 'should',
+ 'show',
+ 'showed',
+ 'showing',
+ 'shows',
+ 'side',
+ 'sides',
+ 'since',
+ 'small',
+ 'smaller',
+ 'smallest',
+ 'so',
+ 'some',
+ 'somebody',
+ 'someone',
+ 'something',
+ 'somewhere',
+ 'state',
+ 'states',
+ 'still',
+ 'such',
+ 'sure',
+ 't',
+ 'take',
+ 'taken',
+ 'than',
+ 'that',
+ 'the',
+ 'their',
+ 'them',
+ 'then',
+ 'there',
+ 'therefore',
+ 'these',
+ 'they',
+ 'thing',
+ 'things',
+ 'think',
+ 'thinks',
+ 'this',
+ 'those',
+ 'though',
+ 'thought',
+ 'thoughts',
+ 'three',
+ 'through',
+ 'thus',
+ 'to',
+ 'today',
+ 'together',
+ 'too',
+ 'took',
+ 'toward',
+ 'turn',
+ 'turned',
+ 'turning',
+ 'turns',
+ 'two',
+ 'u',
+ 'under',
+ 'until',
+ 'up',
+ 'upon',
+ 'us',
+ 'use',
+ 'uses',
+ 'used',
+ 'v',
+ 'very',
+ 'w',
+ 'want',
+ 'wanted',
+ 'wanting',
+ 'wants',
+ 'was',
+ 'way',
+ 'ways',
+ 'we',
+ 'well',
+ 'wells',
+ 'went',
+ 'were',
+ 'what',
+ 'when',
+ 'where',
+ 'whether',
+ 'which',
+ 'while',
+ 'who',
+ 'whole',
+ 'whose',
+ 'why',
+ 'will',
+ 'with',
+ 'within',
+ 'without',
+ 'work',
+ 'worked',
+ 'working',
+ 'works',
+ 'would',
+ 'x',
+ 'y',
+ 'year',
+ 'years',
+ 'yet',
+ 'you',
+ 'young',
+ 'younger',
+ 'youngest',
+ 'your',
+ 'yours',
+ 'z']
+
+
+def words():
```

Comment:
Never mind, makes sense since it allows them to define additional logic for creating the word list.